beancount_import/matching.py

```diff
@@ -255,6 +295,86 @@ def add_posting(self, entry: Transaction, mp: MatchablePosting):
         group = self._postings.setdefault(_date_key(entry, mp), {})
         group[source_posting_ids] = (entry, mp)
 
+        weight = get_posting_weight(mp.posting)
+        if weight is None:
```

Comment:
I think `weight` should never be None for a MatchablePosting

Reply:
See https://github.com/jbms/beancount-import/pull/174#discussion_r1089647813